### PR TITLE
Ownership speaker notes

### PR DESCRIPTION
**Pull request by brandonpollack23**
been sitting on these a minute to collate more, but I saw there are already quite a few in this chapter

### CODE REVIEW

File: src/ownership/move-semantics.md [hunk 1]
```diff
@@ -16,3 +16,11 @@ fn main() {
 * When `s1` goes out of scope, nothing happens: it has no ownership.
 * When `s2` goes out of scope, the string data is freed.
 * There is always _exactly_ one variable binding which owns a value.
+
+<details>
+
+* Mention that this is the opposite of the defaults in C++, which copies by value unless you use std::move (and the move constructor is defined!)
```
Comment (mgeisler) [inline src/ownership/move-semantics.md:22]:
```suggestion
* Mention that this is the opposite of the defaults in C++, which copies by value unless you use `std::move` (and the move constructor is defined!).
```

File: src/ownership/move-semantics.md [hunk 2]
```diff
@@ -16,3 +16,11 @@ fn main() {
 * When `s1` goes out of scope, nothing happens: it has no ownership.
 * When `s2` goes out of scope, the string data is freed.
 * There is always _exactly_ one variable binding which owns a value.
+
+<details>
+
+* Mention that this is the opposite of the defaults in C++, which copies by value unless you use `std::move` (and the move constructor is defined!).
+
+* In rust, you have to be explicity to clone/copy values, this is by using `clone`.
```
Comment (mgeisler) [inline src/ownership/move-semantics.md:24]:
```suggestion
* In Rust, you clones are explicit (by using `clone`).
```

**Comment by mgeisler** [comment]
Thanks @brandonpollack23!

File: src/ownership/move-semantics.md [hunk 3]
```diff
@@ -16,3 +16,11 @@ fn main() {
 * When `s1` goes out of scope, nothing happens: it has no ownership.
 * When `s2` goes out of scope, the string data is freed.
 * There is always _exactly_ one variable binding which owns a value.
+
+<details>
+
+* Mention that this is the opposite of the defaults in C++, which copies by value unless you use `std::move` (and the move constructor is defined!).
+
+* In Rust, you clones are explicit (by using `clone`).
```
Comment (qwandor) [inline src/ownership/move-semantics.md:24]: s/you clones/clones/

**Pull request merged**